gs-test-core/src/main/java/com/avanza/gs/test/InMemoryGigaSpacesManager.java

```diff
@@ -60,6 +69,22 @@ private static boolean shouldStartZooKeeper() {
 		return !Boolean.getBoolean(DISABLE_ZOOKEEPER_PROPERTY);
 	}
 
+	/**
+	 * Sets up temporary GS_HOME in order to avoid files being created in project directory during tests.
+	 */
+	private Path setupGsHome() {
+		if (System.getProperty(CommonSystemProperties.GS_HOME) != null) {
```

Comment:
So if the property is already set, the directory will not be deleted, right? Could maybe be a comment about this.

Reply:
If the property is already set, no directory will be created here and there is nothing to delete (except for the temporary files that are placed in `GS_HOME`, but I think that would be the responsibility of whatever set this property prior to this test execution?)

Reply:
Ok. Yes, I agree. That's not the responsibility of this class. So I guess the use case of setting this property prior to test execution would be that you want to keep the temporary files for some reason. Then maybe that could be the comment here?

Reply:
I added a sentence in the javadoc for this function about this 👍

Reply:
Perfect!